deploy/examples/import-external-cluster.sh

```diff
@@ -65,11 +65,17 @@ function checkEnvVars() {
     echo "Providing both ROOK_EXTERNAL_ADMIN_SECRET and ROOK_EXTERNAL_USER_SECRET is not supported, choose one only."
     exit 1
   fi
+  if [ -n "$KUBECONTEXT" ]; then
+    echo "Using $KUBECONTEXT as the context value for kubectl commands"
+    KUBECTL="kubectl --context=$KUBECONTEXT"
```

Comment:
I see we may do this by `config --kubeconfig=`, not sure what is the right way
https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/

Reply:
generally `--context` is used with kubectl to apply the context for single use

Reply:
Yes, --context is the right one to use when there are multiple clusters within the same kubeconfig.